conf/machine/nezha-allwinner-d1.conf

```diff
@@ -0,0 +1,70 @@
+#@TYPE: Machine
+#@NAME: nezha-allwinner-d1
+#@SOC: Allwinner D1
+#@DESCRIPTION: Machine configuration for Nezha board
+
+require conf/machine/include/riscv/tune-riscv.inc
+
+MACHINE_FEATURES = "screen keyboard ext2 ext3 serial"
+
+KERNEL_CLASSES = "kernel-fitimage"
+KERNEL_IMAGETYPE = "fitImage"
+UBOOT_ENV ?= "boot"
+UBOOT_ENV_SUFFIX = "scr.uimg"
+PREFERRED_PROVIDER_virtual/kernel ?= "linux-nezha-dev"
+PREFERRED_PROVIDER_virtual/bootloader ?= "u-boot-nezha"
+
+PREFERRED_VERSION_openocd-native = "riscv"
+PREFERRED_VERSION_openocd = "riscv"
+
+EXTRA_IMAGEDEPENDS += "opensbi"
+RISCV_SBI_PLAT = "generic"
+
+# Depends on which version of the Nezha board is available, you can choose
+# "512m", "1g" or "2g". It is necessary to point proper Device Tree Blob for
+# U-Boot, OpenSBI or Linux kernel.
+NEZHA_DRAM_SIZE ?= "1g"
```

Comment:
This is an awful hack which will never be merged upstream, and since https://github.com/smaeul/sun20i_d1_spl/commit/525883d3721f4c4d78b498e780b44e85d0676abf it is entirely unnecessary. Please use the DTB that is already provided by U-Boot in RAM (at `$fdtcontroladdr`), and things will Just Work for every board variant.

Reply:
It will be difficult to use `fitImage` in this case as it has a built-in DTB. Maybe then I should use `uImage` format and load the kernel with the DTB loaded from RAM?

Reply:
From what I understand, it should be possible to build and boot a FIT image that does not contain a DTB.